src/Game.cpp

```diff
@@ -348,12 +351,27 @@ void Game::Battle() {
         _Player->DisplayHUD(_Enemy);
         _Enemy->DisplayHUD();
         int damagePlayer = _Player->Action();
+        int extradamagePlayer = _Player->ReturnExtraMolotovDamage();
         // Player's turn to attack Enemy or choose other action.
-        if (damagePlayer != SKIP_TURN) {
+        if (damagePlayer != SKIP_TURN && damagePlayer != -3) {
```

Comment:
Avoid using magic numbers like `-3`, you should define constants e.g. like `SKIP_TURN` so it's easier to read :)